examples/animation/multiple_axes.py

```diff
@@ -0,0 +1,82 @@
+"""
+=======================
+Multiple axes animation
+=======================
+
+This example showcases:
+
+    how animation on subplots works,
+
+    using a figure artist in the animation.
+
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+from matplotlib.patches import ConnectionPatch
+
+fig, (axl, axr) = plt.subplots(
+    ncols=2,
+    sharey=True,
+    figsize=(6, 2),
+    gridspec_kw=dict(width_ratios=[1, 3], wspace=0),
+)
+axl.set_aspect(1)
+axr.set_box_aspect(1 / 3)
+axr.yaxis.set_visible(False)
+axr.xaxis.set_ticks([0, np.pi, 2 * np.pi], ["0", r"$\pi$", r"$2\pi$"])
+
+# draw circle with initial point in left Axes
+x = np.linspace(0, 2 * np.pi, 50)
+axl.plot(np.cos(x), np.sin(x), "k", lw=0.3)
+(point,) = axl.plot(0, 0, "o")
+
+# draw full curve to set view limits in right Axes
+(sine,) = axr.plot(x, np.sin(x))
+
+# draw connecting line between both graphs
+con = ConnectionPatch(
+    (1, 0),
+    (0, 0),
+    "data",
+    "data",
+    axesA=axl,
+    axesB=axr,
+    color="C0",
+    ls="dotted",
+)
+# in order to use blitting, con needs to be added to an Axes (not fig)
```

Comment:
yes, maybe I should better write 

> in order to use blitting, con would need to be added to an Axes (not fig)

what I want to say is that if we added `con` to an Axes we could use blitting, but we don't do it because we want to use just one connection patch that stretches over multiple Axes - hence we must add it as a Figure artist

to avoid confusion, maybe it's better to drop this comment completely?

Reply:
done